src/plugins/vis_type_vega/public/vega_inspector/vega_inspector.scss

```diff
@@ -0,0 +1,13 @@
+.insVegaViewWrapper {
+  height: 100%;
+  > div {
+    height: 100%;
+  }
+  [role="tabpanel"] {
```

Comment:
Please use classes as much as possible so we know exactly what is being targeted.

Reply:
we don't have any class here =( 
![image](https://user-images.githubusercontent.com/20072247/87557550-b6bbc180-c6c0-11ea-8124-0f83b7077227.png)